scripts/prepare_release.sh

```diff
@@ -17,9 +17,8 @@ if [[ ! -z "$(git status -s)" ]]; then
 	exit 2
 fi
 
-# TODO: Update version in CHANGELOG and docs:
-# ## in docs, we require to update _data/sidebars/home_sidebar.yml and _data/topnav.yml
-echo "WARNING: CHANGELOG and docs are not updated!!"
+# TODO: Update version in CHANGELOG
+echo "WARNING: CHANGELOG version is not updated!!"
```

Comment:
Move to the end of the file. And include another WARNING pointing out that the release should include the current jar.